Resources/Prototypes/ADT/Research/industrial.yml

```
@@ -161,6 +161,21 @@
   requiredTech:
   - ADTAdvancedEnergy
 
+- type: technology
+  id: ADTReducer
```

```suggestion
  id: ADTInducer
```
Почему редусер то? Если индсуер он от слова индус